restler/engine/core/request_utilities.py

```diff
@@ -10,6 +10,8 @@
 import uuid
 import types
 import threading
+import importlib
```

Comment:
>importlib [](http://example.com/codeflow?start=7&length=9)

Please use import_utilities

Reply:
changing to -

import importlib.util as import_utilities

Reply:
updating per offline discussion - please disregard above